projects/addon-doc/utils/test/try-parse-markdown.spec.ts

```diff
@@ -42,10 +42,10 @@ ${codeSection}html\n${htmlCode}\n${codeSection}
 @NgModule({
     imports: [
         // ...
-        TuiAvatarModule,
+        TuiAvatar,
```

Comment:
Accidental change?

Reply:
It's a just spec file. I just didn't want to have any mention of the module